docs/proposals/20210607-adding-subcommands-server-join-reset-for-yurtctl.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Adding subcommands server, join and reset for yurtctl
+authors:
+  - '@zyjhtangtang'
+reviewers:
+  - '@Fei-Guo'
+  - '@rambohe-ch'
+  - '@huangyuqi'
+creation-date: 2021-06-07T00:00:00.000Z
+last-updated: 2021-06-07T00:00:00.000Z
+status: provisional
+---
+
+# Adding subcommands `server` `join` and `reset` for yurtctl
+
+## Table of Contents
+
+- [Adding subcommands server, join and reset subcommands for yurtctl](#Adding-subcommands-server-join-and-reset-for-yurtctl)
+  - [Table of Contents](#table-of-contents)
+  - [Glossary](#glossary)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+  - [Proposal](#proposal)
+    - [User Stories](#user-stories)
+    - [Creating OpenYurt cluster with yurtctl](#creating-OpenYurt-cluster-with-yurtctl)
+      - [Parameters for yurtctl server](#parameters-for-yurtctl-server)
+    - [Joining node to OpenYurt cluster](#joining-node-to-OpenYurt-cluster)
+      - [Parameters for yurtctl join](#parameters-for-yurtctl-join)
+    - [Resetting node](#resetting-node)
+  - [Implementation History](#implementation-history)
+
+## Glossary
+
+Refer to the [OpenYurt Glossary](https://github.com/openyurtio/openyurt/blob/master/docs/proposals/00_openyurt-glossary.md).
+
+## Summary
+This proposal add three subcommands `server`, `join` and `reset` for yurtctl. The subcommand `server` can create an all-in-one kubernetes cluster, simultaneously convert the kuberntes cluster to an OpenYurt cluster. The subcommand `join` is used to add a new node to an OpenYurt cluster, including cloud nodes and edge nodes. The subcommand `reset` can restore the node to the state before joining OpenYurt cluster.
```

Comment:
how about rename the subcommand `server` to `init`, so the  experience of yurtctl will be consistent with Kubeadm

Reply:
OK, I have renamed the subcommand server to init.